src/components/SharingSidebarView.vue

```diff
@@ -305,9 +305,12 @@ export default {
 		},
 		toggleAclCreate() {
 			this.showAclCreate = !this.showAclCreate
-			Vue.nextTick(() => {
-				this.$refs.select.$el.focus()
-			})
+			if (this.showAclCreate) {
+				Vue.nextTick(() => {
+					// hacky way to get the Vue-Select instance from the NcSelect
+					this.$refs.select._vnode.componentInstance.open = true
```

Comment:
```suggestion
					this.$refs.select.$el.querySelector('input').focus()
```
This way it will have keyboard focus like before + open the dropdown